README.md

```diff
@@ -103,6 +103,12 @@ Include the following stylesheets in your application:
 /node_modules/react-grid-layout/css/styles.css
 /node_modules/react-resizable/css/styles.css
 ```
+If using Create React App:
+
+```
+import 'react-grid-layout/css/styles.css' 
```

Comment:
what the difference between:

```
Include the following stylesheets in your application:
'''
/node_modules/react-grid-layout/css/styles.css
/node_modules/react-resizable/css/styles.css
'''
```
and your addition for Create React App? 

I think it's not about CRA at all, it's about webpack and importing .css files like modules. In my opinion, we can't describe all possible ways to include those files and it's not so necessary to specify it in documentation.